panther_description/urdf/wheel.urdf.xacro

```diff
@@ -49,10 +49,11 @@
       </visual>
 
       <collision>
-        <origin xyz="0.0 0.0 0.0" rpy="0.0 0.0 0.0" />
+        <origin xyz="0.0 0.0 0.0" rpy="-${pi/2} 0.0 0.0" />
         <geometry>
-          <mesh
-            filename="package://${wheel_config['mesh_package']}/${wheel_config['folder_path']}/wheel_collision.stl" />
+          <cylinder radius="${wheel_config['wheel_radius']}" length="${wheel_config['wheel_width']}"/>
+          <!-- Simulation do not work well with mesh as collision layer. Kept just in case. -->
+          <!-- <mesh filename="package://${wheel_config['mesh_package']}/${wheel_config['folder_path']}/wheel_collision.stl" />  -->
```

Comment:
IMO we can get rid of this comment. Using a simplified representation of a specific part for collision definition is quite natural.